packages/mongodb-chatbot-eval/src/commands/evaluate.ts

```diff
@@ -1,22 +1,38 @@
 import { CommandModule } from "yargs";
 import { LoadConfigArgs, withConfig, withConfigOptions } from "../withConfig";
 import { EvalConfig } from "../EvalConfig";
+import { generateEvalsAndMetadata } from "../evaluate/generateEvalsAndMetadata";
+import { ObjectId } from "mongodb-rag-core";
 
-const commandModule: CommandModule<unknown, LoadConfigArgs> = {
+interface EvaluateCommandArgs {
+  name: string;
+  generatedDataRunId: string;
+}
+
+const commandModule: CommandModule<
+  unknown,
+  LoadConfigArgs & EvaluateCommandArgs
+> = {
   command: "evaluate",
   builder(args) {
     return withConfigOptions(args)
-      .string("name")
-      .option("generatedDataQuery", {
+      .option("name", {
         type: "string",
-        description: "Query to filter generated data.",
+        demandOption: true,
+        description: "Name of the evaluation.",
       })
-      .demandOption("generatedDataQuery")
-      .demandOption("name");
+      .option("generatedDataRunId", {
+        type: "string",
+        description:
+          "RunId for a 'generate' command that you want to create evaluations for. If not provided, uses the `defaultGeneratedDataQuery` from the config. Throws an error if no filter provided here or there's no default.",
```

Comment:
What's the use case for `defaultGeneratedDataQuery`? Seems like you'd always want to have the eval associated with a specific set of generated data and fail if that's not provided. And based on the `demandOption: true` it seems like `generatedDataRunId` should always be provided anyway.

Reply:
ah yes, good catch. that was from a previous implementation that i scrapped b/c it didn't make much sense. let me update this string. 

```suggestion
          "RunId for a 'generate' command that you want to create evaluations for.",
```